lib/mock_transport.ex

```diff
@@ -0,0 +1,101 @@
+defmodule Janus.MockTransport do
+  @moduledoc """
+  Allows to mock transport module in a predictable way.
+
+  One has to pass list of tuples representing request-response pairs to `c:connect/1` callback which will put them in returned state.
+  For each `c:send/3` invocation `#{inspect(__MODULE__)}` will try to match on first occurrence of given request and return
+  corresponding response. Matched tuple is then removed from the list of request-response pairs.
+
+  Module takes into consideration that each request will have `:transaction` field added by `Janus.Connection` module,
+  therefore it will extract `:transaction` field and put it to corresponding response.
```

Comment:
```suggestion
  The module takes into consideration that each request will have `:transaction` field added by `Janus.Connection` module,
  therefore it will extract `:transaction` field and put it to the corresponding response.
```